webcurator-docs/guides/system-administrator-guide.rst

```diff
@@ -73,29 +63,27 @@ Other versions of the required products may be compatible with the Web
 Curator Tool but they have not been tested. Due to the products use of
 Hibernate for database persistence other database platforms should work,
 if the product is rebuilt with the correct database dialect. However
-only Postgesql, Oracle 11g, and MySQL have been tested.
+only Postgesql, Oracle 12c, MySQL and MariaDB have been tested.
 
 Supported platforms
 -------------------
 
 The following platforms have been used during the development of the Web
 Curator Tool:
 
--  Sun Solaris 10
-
 -  Red Hat Linux EL3.
 
 -  Ubuntu GNU/Linux 16.04 LTS
 
--  Windows 7 Ultimate
+-  Windows 7 Ultimate, Windows 10
 
 Other platforms
 ---------------
 
 The following platforms were used during the Development of the Web
 Curator tool but are not explicitly supported:
 
--  Windows 2000, Windows XP Pro, Windows Server 2003
+-  Sun Solaris 10
 
 Optional prerequisites
 ----------------------
```

Comment:
We now also need gradle to build from source. I'm building with version 4.4.1, which is really old, everything newer than that should be fine.